handlers/fb_handlers.php

```diff
@@ -24,18 +24,20 @@
 // and Facebook
 
 // TODO: this only works if the script is installed in root
-define('FACEBOOK_SDK_V4_SRC_DIR', __DIR__ . '/../include/facebook-php-sdk-v4/src/Facebook/');
-require_once(__DIR__ . '/../include/facebook-php-sdk-v4/autoload.php');
+define('FACEBOOK_SDK_V4_SRC_DIR', __DIR__ . '/../include/php-graph-sdk/src/Facebook');
+require_once(__DIR__ . '/../include/php-graph-sdk/src/Facebook/autoload.php');
```

Comment:
Thank you @phelipebf 
As shown [here](https://github.com/teldosas/fbwlan/tree/fbapi-up/include) the submodule `php-graph-sdk ` references facebook/php-graph-sdk@2839246 which is identical to branch 5.4.

[This](http://stackoverflow.com/questions/8813249/git-submodules-specify-a-specific-sha) helped me to better understand git submodules.

Should I add the `branch =5.4` on .gitmodules anyway?